website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -250,3 +250,150 @@ The GET endpoints above can optionally return related resources, if requested wi
 | ------------------ | -------------------------------------- |
 | `run`              | The run this policy check belongs to.  |
 | `run.workspace`    | The associated workspace of the run.   |
+
+
+## List Policy Evaluations in the task stage
+
+`GET /task-stages/:task_stage_id/policy-evaluations`
+
+| Parameter                   | Description                                 |
+| --------------------------- | ------------------------------------------- |
+| `:task_stage_id`            | The task stage ID to get.                   |
+
+This endpoint allows you to list policy evaluations that are part of the task stage.
+
+| Status  | Response                                      | Reason                                              |
+| ------- | --------------------------------------------- | --------------------------------------------------- |
+| [200][] | [JSON API document][]                         | Success                                             |
+| [404][] | [JSON API error object][]                     | Task stage not found                                |
+
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](/cloud-docs/api-docs/#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
+| Parameter                          | Description                                                                                                                                                                                                                                                                                                                 |
+|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `page[number]`                     | **Optional.** If omitted, the endpoint will return the first page.                                                                                                                                                                                                                                                          |
+| `page[size]`                       | **Optional.** If omitted, the endpoint will return 20 agent pools per page.                                                                                                                                                                                                                                                 |
```

Comment:
Just removing "will" --> We want to describe TFC features and actions in present tense. 

```suggestion
| `page[number]`                     | **Optional.** If omitted, the endpoint returns the first page.                                                                                                                                                                                                                                                          |
| `page[size]`                       | **Optional.** If omitted, the endpoint returns 20 agent pools per page.                                                                                                                                                                                                                                                 |
```